gm4_boots_of_ostara/data/gm4_boots_of_ostara/function/convert.mcfunction

```diff
@@ -0,0 +1,21 @@
+# checks what the player/armor stand is holding
+# @s = player or armor stand, wearing Boots of Ostara
+# at @s
+# run from main
+
+# make sure moss/mycelium checks are run before grass, so grass isn't instantly converted to either
+# this rule replaces the old "check if grass was converted" check
+
+# first run, check mainhand
+execute if predicate gm4_boots_of_ostara:mainhand_moss if block ~ ~-0.5 ~ #gm4_boots_of_ostara:moss_replaceable run setblock ~ ~-0.5 ~ moss_block
```

Comment:
Instead of having all these predicates in separate files, I think it would be more readable to make use of `if items` here

Reply:
Resolved with 5606fbc